src/Row.tsx

```diff
@@ -150,7 +148,6 @@ const Row: BsPrefixRefForwardingComponent<'div', RowProps> = React.forwardRef<
         className={classNames(
           className,
           decoratedBsPrefix,
-          noGutters && 'no-gutters',
```

Comment:
Need to remove `noGutters` from the TS interface above as well.  Also don't forget to fix the warnings after you're done.

Reply:
Done! And sorry, I merged master to see if it would fix CI fails